build/source/driver/multi_driver.f90

```diff
@@ -211,23 +211,44 @@ program multi_driver
 integer(i4b)                     :: err=0                      ! error code
 character(len=1024)              :: message=''                 ! error message
 
+! subsetting the model run
+integer(i4b)                     :: strtHRU                    ! index of the starting hru for a subset run
```

Comment:
Can we please have more descriptive name. For example, `startHRU` instead of `strtHRU`